serverless/architecture/serverless-event-architecture.adoc

```diff
@@ -0,0 +1,33 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-event-architecture"]
+= Knative Eventing architecture
+:context: serverless-event-architecture
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+Knative Eventing on {product-title} enables developers to more easily declare how components of their system communicate, using an link:https://www.redhat.com/en/topics/integration/what-is-event-driven-architecture[event-driven architecture] for serverless applications.
+Event-driven architecture is based on the concept of decoupled relationships between event producers and event  _sinks_, or consumers, that receives those events.
+
+You can create a fan out event delivery system from an xref:../event_sources/knative-event-sources.adoc#knative-event-sources[event source] to multiple event sinks using:
+
+* xref:../knative_eventing/serverless-channels.adoc#serverless-channels[Channels] and Subscriptions, or
+* xref:../knative_eventing/serverless-using-brokers.adoc#serverless-using-brokers[Brokers] and xref:../knative_eventing/serverless-kn-trigger.adoc#serverless-kn-trigger[Triggers].
+// TODO: Add subscription docs
+
+The Channel or Broker implementation manages delivery of events to the requested sinks, through either Subscriptions or Triggers, and will buffer events if the destination sink is unavailable. Using the broker and trigger together enables an event delivery mechanism that hides the details of event routing from the event producer and event consumer.
```

Comment:
Should broker and trigger be lower case here?

Reply:
No, I don't think so. They're a `kind` the same way Service, Event, Channel etc are.